vyper/compiler/output.py

```diff
@@ -136,6 +142,14 @@ def build_interface_output(compiler_data: CompilerData) -> str:
                 out += f"    {member_name}: {member_type}\n"
             out += "\n\n"
 
+    if len(interface.flags) > 0:
+        out += "# Flags\n\n"
+        for flag in interface.flags.values():
+            out += f"flag {flag.name}:\n"
+            for flag_value in flag._flag_members:
+                out += f"    {flag_value}\n"
+            out += "\n\n"
```

Comment:
@cyberthirst for future reference, maybe this should have been oos? as it was not in the original scope and could be implemented in a follow up PR